runtime/vm/exceptionsupport.c

```diff
@@ -519,6 +519,16 @@ internalSetCurrentExceptionWithCause(J9VMThread *currentThread, UDATA exceptionN
 	currentThread->currentException = NULL;
 	currentThread->privateFlags &= ~J9_PRIVATE_FLAGS_REPORT_EXCEPTION_THROW;
 
+	if (J9VMCONSTANTPOOL_JAVALANGUNSUPPORTEDCLASSVERSIONERROR == exceptionNumber) {
```

Comment:
should this be `index` instead of `exceptionNumber` as per the stackoverflow case below

Reply:
Agree, fixed. Please review again.